packages/web3-plugin-wallet-rpc/src/WalletRpcPlugin.ts

```diff
@@ -64,8 +83,9 @@ export class WalletRpcPlugin extends Web3PluginBase<WalletRpcApi> {
    *
    * See [EIP-2015](https://eips.ethereum.org/EIPS/eip-2015) for more details.
    *
-   * @param param - Details of the chain to switch to and possibly add
-   * @returns a Promise that resolves if the request is successful
+   * @param param - Details of the chain to switch to and possibly add.
+   * @returns A Promise that resolves if the request is successful.
+   * @experimental
```

Comment:
Since other libraries don’t implement this method and wallets appear not to support them, I’ve moved it to experimental. I’ll eventually remove it if I don’t find a wallet that supports it.